carina-webdriver/src/main/java/com/qaprosoft/carina/core/foundation/webdriver/DriverHelper.java

```diff
@@ -396,17 +396,12 @@ public void openURL(String url) {
         }
         
         try {
-    		if ("chrome".equalsIgnoreCase(Configuration.get(Parameter.BROWSER))) {
-    			driver.manage().window().setSize(new Dimension(1920, 1040));
-    		} else {
-    			driver.manage().window().maximize();
-    		}
+            driver.manage().window().maximize();
```

Comment:
I remember chrome had a bug with maximize. Is it fixed now? what's chrome/selenium did you test?

Reply:
I was confident that this was a workaround for a bug.
Tested on Chrome 70 Selenium as configured in pom.xml (3.141.59)
I think it was fixed a while ago